.changeset/mean-gifts-know.md

```diff
@@ -0,0 +1,81 @@
+---
+'astro': minor
+'@astrojs/vercel': patch
+---
+
+The responsive images feature introduced behind a flag in [v5.0.0](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md#500) is no longer experimental and is available for general use.
+
+The `experimental.responsiveImages` flag has been removed, and all experimental image configuration options have been renamed to their final names.
+
+## Migration Guide
```

Comment:
Before this, don't forget this is now hype time for this "new feature"! Feel free to pull in stuff from the original experimental PR, describing what responsive images are, why they're cool etc.!